app/src/main/java/com/pusher/demo/features/marketplace/chat/MarketplaceChatPresenter.kt

```diff
@@ -109,13 +109,32 @@ class MarketplaceChatPresenter :  BasePresenter<MarketplaceChatPresenter.View>()
     }
 
     private fun getOtherMemberReadCursor(room: Room, otherMember : User) {
-        val readCursor = currentUser().getReadCursor(room, otherMember).successOrThrow()
-        view?.onOtherMemberReadCursorChanged(readCursor.position)
+        currentUser().getReadCursor(room, otherMember).fold(
+            onFailure = {
+                // this situation can happen when the room has just been created, and the other
+                // persons cursor has not yet been created as they have not yet read any messages.
+                Log.d(ChatkitManager.LOG_TAG, "failed to create the other members cursor")
```

Comment:
It's not failure though, just absence of a cursor (normal case). Log level debug is fine but the message should match it.

```suggestion
                Log.d(TAG, "other member's cursor not set yet")
```

I think then we won't need that long (moreover currently repeated) comment.

Oh and also it's worth following the convention of setting TAG per file — some devs may be jumping straight into a file and search there for a static part of a message as opposed to using the global search.

Reply:
Okay! That makes sense. I've updated the log messages (and added the TAG per file) - I've kept the comments as it let's people know how this situation arrises. Does that still make sense?